### ISSUE TITLE
instruction for testing

### ISSUE THREAD

**Issue by rokopi-byte** [dongkwonjin/Semantic-Line-MWCS#6]
Hi, could you provide basic instruction to test the code ? I'd like to try inference on my images. It is my understanding that I have to preprocess my images using Preprocessing/main.py but is not clear to me how to edit the option/config.py file.

**Comment by dongkwonjin** [comment]
Hi, thank you for your attention!

1) For testing on your own dataset, you have to use the files in Modeling/H_Net/code. You don't need to run the codes in Preprocessing/, because they are used to generate training data and line candidates. You can find 'candidates.pickle' file by downloading preprocessed data. 
2) We recommend using a dataloader for testing. To this end, you need to make a file in Modeling/H_Net/code/datasets/, similar to dataset_sel.py or dataset_sl5k.py file. In your dataset_YOURS.py file, you load your image and 'candidates.pickle' file.
3) Then, in option/config.py, set self.run_mode='test', self.dataset_name = 'YOURS', and add your image path to self.dir['dataset']['YOURS']. Also, you have to add some lines in Modeling/H_Net/code/libs/prepare.py file.
4) Run main.py file.

**Comment by msanchezvicom** [comment]
Hi,

@rokopi-byte Could you finally try inference in your own dataset?

@dongkwonjin Congrats on your job! I would like to test it in my own dataset but it is hard to follow the testing instructions. Please, could you provide further information? Where can I get the candidates.pickle file?

Thank you very much
